web-registry/src/components/organisms/BasketEcocreditsTable/BasketEcocreditsTable.tsx

```diff
@@ -63,38 +59,42 @@ export const BasketEcocreditsTable: React.FC<
         <BreakText>Batch End Date</BreakText>,
         'Project Location',
       ]}
-      rows={batches.map(item => {
-        const projectCell =
-          item.projectHandle === '-' ? (
-            item.projectName
-          ) : (
-            <Link href={`/project/${item.projectHandle}`} target="_blank">
-              {item.projectName}
-            </Link>
-          );
-
-        return [
-          projectCell,
-          <Box
-            component="span"
-            sx={{
-              whiteSpace: {
-                xs: 'wrap-word',
-                md: 'nowrap',
-              },
-            }}
+      rows={basketCredits.map(credit => [
+        <WithLoader isLoading={credit.projectName === ''} variant="skeleton">
+          <Link
+            href={`/project/${credit.projectId}`}
+            target="_blank"
+            sx={tableStyles.ellipsisColumn}
+          >
+            {credit.projectName}
+          </Link>
+        </WithLoader>,
+        <WithLoader isLoading={!credit.denom} variant="skeleton">
+          <Link href={`/credit-batches/${credit.denom}`}>{credit.denom}</Link>
+        </WithLoader>,
+        <AccountLink address={credit.issuer} />,
+        formatNumber({
+          num: credit.balance,
+          ...quantityFormatNumberOptions,
+        }),
+        <WithLoader isLoading={credit.classId === ''} variant="skeleton">
+          <Link
+            key="class_id"
```

Comment:
just curious - why is the `key` necessary?

Reply:
It's not, I took this field from the ecocredits table and did not see that. Will delete it there as well.